newspeak/KernelForPrimordialSoup.ns

```diff
@@ -506,6 +518,9 @@ public isEmpty ^<Boolean> = (
 public isKindOfCollection ^<Boolean> = (
 	^true
 )
+newForCollectUsingAdd: capacity = (
```

Comment:
IWhy? Because it is a subclass responsibility. The method is used within the Collection class. If anyone ever subclasses it, they need to implement this.  Sure, they could get a MNU and figure it out, but if they get subclass responsibility, it's easier. It is also documented in the code and searchable. And, if one implements a typechecker, it helps to0.